nav2_amcl/src/amcl_node.cpp

```diff
@@ -1543,6 +1553,10 @@ AmclNode::initServices()
     "reinitialize_global_localization",
     std::bind(&AmclNode::globalLocalizationCallback, this, _1, _2, _3));
 
+  initial_guess_srv_ = create_service<nav2_msgs::srv::SetInitialPose>(
+    "set_initial_pose",
+    std::bind(&AmclNode::initialGuessCallback, this, _1, _2, _3));
```

Comment:
This does not compile, that function does not exist anymore.

Please test your changes

Reply:
Now it compiles again. At least on my computer